public/index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="viewport" content="width=device-width, initial-scale=1,0" />
```

Comment:
Should be a dot instead of a comma
```suggestion
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Reply:
Done